### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by joshmoore**
* Upgrade to 3.7.4
 * Drop beta from EBI hostname
 * Transfer README from Gitlab
 * Add a quick test to .travis.yml

Propose 0.2.0 or _perhaps_ 1.0.0.

A major goal of this PR is to have travis green and then enable a cron job. cc: @sbesson @manics

### CODE REVIEW

**Comment by joshmoore** [comment]
Re-opening to trigger travis.

**Comment by joshmoore** [comment]
This passed locally:

```
/home/data/.aspera/connect/bin/ascp -P33001 -l 40m -i /home/data/.aspera/connect/etc/asperaweb_id_dsa.openssh idr0001@fasp.ebi.ac.uk:md5sum.txt /tmp/idr0001/
```

Not sure if ports are being blocked by travis.

**Comment by joshmoore** [comment]
Re-run went green, which doesn't fill me with confidence about testing via travis. Nevertheless, I've enabled cron (daily unless a build within the past 24 hours).

File: README.md [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
+The https://hub.docker.com/r/imagedata/download image can be used to acquire
+the original files which were imported into IDR:
+
+        docker run -ti --rm -v /tmp/data:/data imagedata/download idr0001 20151116-verified/JL_120731_S6A /data/JL_120731_S6A
+
+Some exemplary source directories are listed below:
+
+| study   | source                                                                                                      | plate                                       | size   |
+|---------|-------------------------------------------------------------------------------------------------------------|---------------------------------------------|--------|
+| idr0001 | 20151116-verified/JL_120731_S6A                                                                             | JL_120731_S6A                               | # 99G  |
```
Comment (manics) [inline README.md]: Are the `#`s in front of the sizes meant to indicate something?

**Review by manics — COMMENTED** [review]
One comment on the readme, otherwise looks fine.
`docker run -it -v /tmp/data:/data --rm idr-aspera-client-docker idr0027 ./ /data` recursively downloads the whole of idr0027 (though I stopped it after 1G).

File: README.md (hunk 1 above)
Comment (joshmoore) [reply README.md]: I assume they were a leftover of sloppy `vim` work. Pushed a removal.

**Review by manics — APPROVED** [review]

**Comment by joshmoore** [comment]
Merging and releasing as 0.2.0 to activate cron.

**Pull request merged**